examples/bluesky/main.py

```
@@ -154,17 +168,48 @@ async def _fetch_author(self, handle: str, cursor: str | None) -> None:
+        text_w = pane_w - TEXT_X - PAD
+
+        heights = []
+        for post in self._feed:
+            body = _text(post)
+            if body:
+                try:
+                    h = await self.emit.measure_text_wrapped(
+                        body, CAPTION, text_w, max_lines=MAX_LINES
+                    )
+                    heights.append(ROW_BASE + max(0.0, h - CAPTION))
+                except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)